vector/src/main/java/im/vector/app/features/roomprofile/uploads/media/RoomUploadsMediaFragment.kt

```diff
@@ -82,15 +82,18 @@ class RoomUploadsMediaFragment :
         controller.listener = this
     }
 
-    @Suppress("DEPRECATION")
     private fun getNumberOfColumns(): Int {
-        val displayMetrics = DisplayMetrics()
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
-            requireContext().display?.getMetrics(displayMetrics)
+        val screenWidthInPx = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
+            val a = requireActivity().windowManager.currentWindowMetrics
+            a.bounds.width()
```

Comment:
Nit: I don't think the variable serves any value here

Reply:
yes, it's assigned to `val screenWidthInPx`.